projects/storefrontapp/src/app/spartacus/features/organization/organization-registration-wrapper.module.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule, Type } from '@angular/core';
+import { CDCB2BRegisterModule } from '@spartacus/cdc/organization/user-registration';
+import { OrganizationUserRegistrationModule } from '@spartacus/organization/user-registration';
+
+import { environment } from '../../../../environments/environment';
+
+const extensions: Type<any>[] = [];
+
+if (environment.cdc) {
+  extensions.push(CDCB2BRegisterModule);
+}
+
+@NgModule({
+  imports: [OrganizationUserRegistrationModule, ...extensions],
```

Comment:
Comment related not to this file, but to this PR:

This PR is missing a schematics configuration that would import `CDCB2BRegisterModule` inside a wrapper modules in customer's applications.

Please follow the example of e.g. `CdcAdministrationModule`.

For more:
- see the [existing schematics config for cdc](https://github.com/SAP/spartacus/blob/95a7347f51ea1e3aae3dcade4c01d0f8714da983/projects/schematics/src/shared/lib-configs/integration-libs/cdc-schematics-config.ts#L44)
- see [schematics configuration docs](https://github.com/SAP/spartacus/blob/95a7347f51ea1e3aae3dcade4c01d0f8714da983/projects/schematics/README.md#creating-and-configuring-feature-schematics)
- contact your team collegues who already added schematics

Before the merge, please also test that your schematics correctly install and import your new module in a fresh Angualr app, while running Spartacus installation schematics. And test that your feature works correctly in a fresh Angular app with Spartacus installed via schematics. (It's a part of DoD)